### PR TITLE
Added sanctuary-zero to Projects

### PR DESCRIPTION
**Pull request by t0xic0der**

### CODE REVIEW

**Comment by jonathanslenders** [comment]
Thanks!

File: PROJECTS.rst [hunk 1]
```diff
@@ -46,6 +46,7 @@ Full screen applications:
 - `pypager <https://github.com/prompt-toolkit/pypager>`_: A $PAGER in pure Python (like "less").
 - `kubeterminal <https://github.com/samisalkosuo/kubeterminal>`_: Kubectl helper tool.
 - `pydoro <https://github.com/JaDogg/pydoro>`_: Pomodoro timer.
+- `sanctuary-zero <github.com/t0xic0der/sanctuary-zero>`_: A secure chatroom with zero logging and total transience
```
Comment (bjorn3) [inline PROJECTS.rst:49]: This is missing the `https://` prefix. It is a broken link right now.

**Comment by jonathanslenders** [comment]
Very nice project, by the way!

File: PROJECTS.rst (hunk 1 above)
Comment (jonathanslenders) [reply PROJECTS.rst:49]: Right, I'll fix that.

Comment (jonathanslenders) [reply PROJECTS.rst:49]: Fixed here: https://github.com/prompt-toolkit/python-prompt-toolkit/commit/551a5b8fb4901ea24dc334da707e84bcf0ddc6d0 Thanks for pointing out!

**Pull request merged**